airflow-core/src/airflow/jobs/scheduler_job_runner.py

```diff
@@ -1923,43 +1926,26 @@ def _create_dag_runs_asset_triggered(
             dag_run.consumed_asset_events.extend(asset_events)
             session.execute(delete(AssetDagRunQueue).where(AssetDagRunQueue.target_dag_id == dag_run.dag_id))
 
-    def _should_update_dag_next_dagruns(
+    def _finished_and_automated(
         self,
         dag: SerializedDAG,
-        dag_model: DagModel,
         *,
         last_dag_run: DagRun | None = None,
-        active_non_backfill_runs: int | None = None,
-        session: Session,
     ) -> bool:
         """Check if the dag's next_dagruns_create_after should be updated."""
```

Comment:
This docstring should probably be updated to match the func name?

Reply:
yup thanks